spellcheck, corrections, rewording and small extension of the documen… (#107)
Kixunil/cryptoanarchy-deb-repo-builder: docs/user-level.md

## What's the point

> The work of any operating system depends on so called "services", programmes that are run in the background and control many of the functions of the operating systems and other programs. Those are the programs that provide interface between the hardware and software (called drivers), index files for quick search of your documents, manage the way your audio is mixed before it gets to your speakers and many others. 

Author: Sure, feel free to use just what you feel is appropriate.

You don't need to know many of the system services work and which programs depend on them, but it's important that their presence is vital for the system. In a popular GNU/Linux distributions such as Debian rely on a package manager that automatically figures out which services need to be installed and configured whenever you choose a new application to be installed on your computer.

> We believe that this is the way bitcoin and cryptocurrency related software should behave as well, but the current situation is different. The user is often referred to a github repository, where in the described installation is tedious manual procedure prone to errors and security holes arising from misconfiguration or failure to verify the software properly. 

Reviewer: ```suggestion
We believe that this is the way bitcoin and cryptocurrency related software should behave as well, but the current situation is different. The user is often referred to a GitHub repository, where in the described installation is tedious manual procedure prone to errors and security holes arising from misconfiguration or failure to verify the software properly. 
```

> This repository was made to address this issue and allow to install cryptocurrency software stack automatically. It aims to create the required package files installable with Debian package manager. 

Reviewer: ```suggestion
This repository was made to address this issue and allow to install a Bitcoin software stack automatically. It aims to create the required package files installable with Debian package manager. 
```

We believe that this is the way bitcoin and cryptocurrency related software should behave as well, but the current situation is different. The user is often referred to a github repository, where in the described installation is tedious manual procedure prone to errors and security holes arising from misconfiguration or failure to verify the software properly. 

This repository was made to address this issue and allow to install cryptocurrency software stack automatically. It aims to create the required package files installable with Debian package manager. 

Yes, installing any Bitcoin/freedom-related application is now going to be as easy as installing VLC. It'll be possible to use GUI or simple `apt` command.

## What you need to know before installing anything from this repository

(TL;DR below)

**The project is still work in progress!** For the time-being a basic admin skills going to be of a great help. It is often faster and more illustrative to provide the code via commands that you paste to your terminal window, than writing the instruction for GUI. 

> Also, any feedback you might provide is appreciated. You can do so by opening an issue on our github repository page.

Reviewer: ```suggestion
Also, any feedback you might provide is appreciated. You can do so by opening an issue on our GitHub repository page.
```

#### Usage

* **Installing `lnd` automatically installs `lnd-unlocker`, which creates a wallet right after installation and stores the seed in `/var/lib/lnd-system-mainnet/.seed.txt`**

> * The above behaviour can be overwritten by `--no-install-recommends`, but **do not do it unless you prefer false sense of security. Without unlocker `lnd` will stay down after crash. If long enough others may steal your sats!**

Reviewer 2: I use US English and it'd make sense to keep it consistent across repository.
I believe "overridden" is more appropriate than "overwritten" when referring to behavior.

Author: Ok, I'm more used to UK spelling. I will push a spell check with us dictionary.
